examples/openai-lander/evolve.py

```diff
@@ -86,21 +86,22 @@ def __init__(self, num_workers):
     def simulate(self, nets):
         scores = []
         for genome, net in nets:
-            observation = env.reset()
+            observation_init_vals, observation_init_info = env.reset()
             step = 0
             data = []
             while 1:
                 step += 1
                 if step < 200 and random.random() < 0.2:
                     action = env.action_space.sample()
                 else:
-                    output = net.activate(observation)
+                    output = net.activate(observation_init_vals)
```

Comment:
Isn't this wrong? Shouldn't you have named this `observation`? Now it just feeds the initial observation every time through the loop, and the observation never changes. Same issue below!

Reply:
i think you are right. I created another PR ... maybe have a loot at it and feel free to comment if u find something
https://github.com/CodeReclaimers/neat-python/pull/274